### PR TITLE
SCALRCORE-16990 [provider] IACP API > Replace ingress-attributes with vcs-revision

### PR DESCRIPTION
**Pull request by penja**
### Changelog
* [x] I have updated CHANGELOG.md

### Documentation
* [ ] I have updated resource/data source documentation in [website/docs](https://github.com/Scalr/terraform-provider-scalr/tree/develop/website/docs)

### State
* [ ] My changes affect the state
* [ ] I have included a state migration and a unit test for it

### CODE REVIEW

File: scalr/data_source_current_run.go [hunk 1]
```diff
@@ -132,21 +132,14 @@ func dataSourceScalrCurrentRunRead(d *schema.ResourceData, meta interface{}) err
...
-			}
-			return fmt.Errorf("Error retrieving ingress attributes: %v", err)
-		}
+		log.Printf("[DEBUG] Read vcs revision attributes of run: %s", runID)
 
 		var commitConfig []map[string]interface{}
 		commit := map[string]interface{}{
-			"sha":     ingressAttributes.CommitSha,
-			"message": ingressAttributes.CommitMessage,
+			"sha":     run.VcsRevision.CommitSha,
```
Comment (mermoldy) [inline scalr/data_source_current_run.go:139]: @penja, the run may not have a `VcsRevision`.

**Review by mermoldy — CHANGES_REQUESTED** [review]

File: scalr/data_source_current_run.go (hunk 1 above)
Comment (penja) [reply scalr/data_source_current_run.go]: @mermoldy fixed

**Review by mermoldy — APPROVED** [review]

File: scalr/data_source_current_run.go [hunk 2]
```diff
@@ -132,30 +132,25 @@ func dataSourceScalrCurrentRunRead(d *schema.ResourceData, meta interface{}) err
...
+		if run.VcsRevision != nil {
+			var commitConfig []map[string]interface{}
+			commit := map[string]interface{}{
+				"sha":     run.VcsRevision.CommitSha,
+				"message": run.VcsRevision.CommitMessage,
+				"author": map[string]interface{}{
+					"username": run.VcsRevision.SenderUsername,
+				},
+			}
+			vcs["commit"] = append(commitConfig, commit)
```
Comment (lbn) [inline scalr/data_source_current_run.go:151]: You can also do `"commit": []map[string]interface{}` in vcs and append to it inside the if statement. This approach would be more consistent with the rest of the provider.

File: scalr/data_source_current_run.go [hunk 3]
```diff
@@ -132,30 +132,25 @@ func dataSourceScalrCurrentRunRead(d *schema.ResourceData, meta interface{}) err
...
 			"branch":        workspace.VCSRepo.Branch,
-			"commit":        append(commitConfig, commit),
 		}
+
+		if run.VcsRevision != nil {
+			var commitConfig []map[string]interface{}
+			commit := map[string]interface{}{
+				"sha":     run.VcsRevision.CommitSha,
+				"message": run.VcsRevision.CommitMessage,
+				"author": map[string]interface{}{
```
Comment (lbn) [inline scalr/data_source_current_run.go:147]: Need go fmt.

File: CHANGELOG.md [hunk 4]
```diff
@@ -7,6 +7,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Changed
+
+- `data.scalr_current_run` requires scalr server >= `8.0.1-beta.CURRENT_RELEASE_VERSION`
```
Comment (lbn) [inline CHANGELOG.md:12]: Needs its own required section.

**Review by lbn — CHANGES_REQUESTED** [review]

File: scalr/data_source_current_run.go (hunk 3 above)
Comment (penja) [reply scalr/data_source_current_run.go:148]: @lbn This one is already formated by `go ftm`

File: scalr/data_source_current_run.go (hunk 2 above)
Comment (penja) [reply scalr/data_source_current_run.go:152]: fixed

**Review by lbn — APPROVED** [review]

**Pull request merged**